.gitignore

```diff
@@ -6,3 +6,22 @@
 *.njsproj
 *.sln
```

Comment:
Why file endings for Visual Studio but not for VS Code or Goland?
Either remove those or also add VSCode and GoLand ignores.